src/components/EmojiPicker/EmojiPicker.js

```diff
@@ -161,7 +172,10 @@ class EmojiPicker extends React.Component {
                     width: CONST.EMOJI_PICKER_SIZE.WIDTH,
                     height: CONST.EMOJI_PICKER_SIZE.HEIGHT,
                 }}
+                outerStyle={Browser.isMobile() && {maxHeight: this.props.windowHeight, marginTop: this.props.viewportOffsetTop}}
```

Comment:
@perunt Let's move `{maxHeight: this.props.windowHeight, marginTop: this.props.viewportOffsetTop}` to a helper functions in `styles.js`.